custom/enikshay/management/commands/data_dumps_person_case.py

```diff
@@ -44,6 +44,15 @@ def get_last_episode(self, case):
             return Exception("could not find last episode for person %s" % case.case_id)
         return self.context['last_episode']
 
+    def get_last_occurrence(self, case):
+        self.context['last_occurrence'] = (
+            self.context.get('last_occurrence') or
+            get_last_occurrence(case)
+        )
+        if not self.context['last_occurrence']:
+            return Exception("could not find last occurrence for person %s" % case.case_id)
```

Comment:
This should be `raise`d

Reply:
oh no.. dont understand why i keep doing this.. sorry.. will update..